keyboards/otaku_split/ver1/keymaps/via/rule.mk

```
@@ -0,0 +1,2 @@
+
+VIA_ENABLE=yes
```

```suggestion
VIA_ENABLE = yes
```
This file must be named `rules.mk`.